### DISCUSSION TITLE
Nous détestons.

### DISCUSSION THREAD

**Discussion by chateaudefrance** [woocommerce/woocommerce#46772] [Checkout Flow]
On ne peut pas imaginer un système pire que ça. Les blocs sur wordpress, c'est épouvantable, très mal conçu. Voulez-vous régler la taille des polices ? Insérer des éléments basiques ? Placer un bloc à côté de l'autre ? Vous ne le pouvez pas, sauf à réessayer dix fois, à rafraîchir la page, vous apercevoir que des modifications ont été prises en compte alors qu'elles étaient erronées. Une horreur. NOUS DETESTONS !

**Comment by senadir** [comment]
How can we make your experience better? can you provide exactly what happened? Because I can change the font size fine and add block just fine.
